ydb/docs/en/core/security/audit-log.md

```diff
@@ -64,16 +322,78 @@ audit_config:
-| `stderr_backend` | Forward the audit log to the standard error stream (`stderr`).</ul>Optional. |
+### Audit config parameters {#audit-config}
+
+All fields are optional.
+
+#|
+|| Field                    | Description ||
+|| `stderr_backend`         | Forward the audit log to the standard error stream (`stderr`). See the [backend settings](#backend-settings) for details. ||
+|| `file_backend`           | Write the audit log to a file at each cluster node. See the [backend settings](#backend-settings) for details. ||
+|| `unified_agent_backend`  | Stream the audit log to the [Unified Agent](https://yandex.cloud/docs/monitoring/concepts/data-collection/unified-agent/). In addition, you need to define the `uaclient_config` section in the [cluster configuration](../reference/configuration/index.md). See the [backend settings](#backend-settings) for details. ||
+|| `log_class_config`       | An array of audit rules for different log classes. See the [log class configuration](#log-class-config). ||
+|| `heartbeat`              | Optional heartbeat configuration. See the [heartbeat settings](#heartbeat-settings). ||
+|#
+
+### Backend settings {#backend-settings}
+
+Each backend supports the following fields:
+
+#|
+|| Field                | Description ||
+|| `format`             | Audit log format. The default value is `JSON`. See [Log format](#log-format) for details.<br/>*Optional.* ||
+|| `file_path`          | Path to the file that the audit log will be streamed to. If the path and the file are missing, they will be created on each node at cluster startup. If the file exists, the data will be appended to it. Only for `file_backend`. <br/>*Required.* ||
+|| `log_name`           | The session metadata delivered with the message. Using the metadata, you can redirect the log stream to one or more child channels based on the condition: `_log_name: "session_meta_log_name"`. Only for `unified_agent_backend`. <br/>*Optional.* ||
+|| `log_json_envelope`  | JSON template that wraps each log record. The template must contain the `%message%` placeholder, which is replaced with the serialized audit record. See the [Envelope format](#envelope-format).</br>*Optional.* ||
+|#
+
+#### Log format {#log-format}
+
+The `format` field specifies the serialization format for audit events. The supported formats are:
+
+#|
+|| Format                 | Description ||
+|| `JSON`                 | Each audit event is serialized as a single-line JSON object preceded by an ISO 8601 timestamp.</br>Example: `<time>: {"k1": "v1", "k2": "v2", ...}` </br>*`k1`, `k2`, …, `kn` represent audit log attributes; `v1`, `v2`, …, `vn` represent their values.* ||
+|| `TXT`                  | Each audit event is serialized as a single-line text string in the `key=value` format, preceded by an ISO 8601 timestamp.</br>Example: `<time>: k1=v1, k2=v2, ...` </br>*`k1`, `k2`, …, `kn` represent audit log attributes; `v1`, `v2`, …, `vn` represent their values.* ||
```

Comment:
What happens with `=`, `,` and ` ` symbols within the values?
And explicitly state what the separator is: comma, and then space or tab?